agent/src/main/java/com/newrelic/agent/android/NewRelic.java

```diff
@@ -1061,8 +1106,123 @@ public static boolean recordJSErrorException(StackTrace stackTrace) {
         return DataController.sendAgentData(stackTrace);
     }
 
-
-    public static boolean  addHTTPHeadersTrackingFor(List<String> headers) {
+    /**
+<<<<<<< HEAD
+     * Adds set of names to network request header instrumentation
+     *
+     * @param headers
+     * @return true
+=======
+     * Adds a set of request header instrumentation targets
+     *
+     * @param headers
+>>>>>>> feature/nr169916-loggingConfiguration
+     */
```

Comment:
merge comments here

Reply:
Thanks Ying, I see that snuck through only because it was part off a comment.